### PR TITLE
chore: update docs guide

### PR DESCRIPTION
**Pull request by vannyle**
This pull request brings the update for the Documentation README, introducing new components for doc pages
- Admonition
- Code Tabs

https://github.com/neondatabase/website/blob/29f4292d5f11d2751d5bbf22837adc7b3f6359fd/content/docs/README.md

### CODE REVIEW

**Review by danieltprice — APPROVED** [review]
LGTM. Thank you!

File: content/docs/README.md [hunk 1]
```diff
@@ -143,6 +167,86 @@ To add a single page <https://example.com/changelog> to the docs sidebar, add th
...
+Example:
+
+```md
+<Admonition type="note" title="Your title">
+The branch creation process does not increase load on the originating project. You can create a branch at any time without worrying about downtime or performance degradation.
+</Admonition>
+<Admonition type="info">
+The branch creation process does not increase load on the originating project. You can create a branch at any time without worrying about downtime or performance degradation.
+</Admonition>
+```
```
Comment (bolotskydev) [inline content/docs/README.md:241]:
```suggestion
```md
<Admonition type="note" title="Your title">
  The branch creation process does not increase load on the originating project. You can create a branch at any time without worrying about downtime or performance degradation.
</Admonition>

<Admonition type="info">
  The branch creation process does not increase load on the originating project. You can create a branch at any time without worrying about downtime or performance degradation.
</Admonition>
```
```

File: content/docs/README.md [hunk 2]
```diff
@@ -143,6 +167,86 @@ To add a single page <https://example.com/changelog> to the docs sidebar, add th
...
+<details>
+<summary>Examples</summary>
+
+![Code tabs example](code-tabs-example.jpg)
+
+</details>
+
+## Admonition
+
+In addition to emphasize your text, wrap text with `<Admonition></Admonition>` and set the proper type.
```
Comment (bolotskydev) [inline content/docs/README.md:226]: To improve the documentation readability, one can leverage an `Admonition` custom component. Just wrap your piece of text with `<Admonition></Admonition>` and pass the type.

File: content/docs/README.md [hunk 3]
```diff
@@ -143,6 +167,86 @@ To add a single page <https://example.com/changelog> to the docs sidebar, add th
...
+
+There are 5 types of Admonition: `note`, `important`, `tip`, `warning`, `info`.
```
Comment (bolotskydev) [inline content/docs/README.md:228]:
```suggestion
There are 5 types of Admonition: `note`, `important`, `tip`, `warning`, `info`; the default is `note`.
```

**Review by bolotskydev — CHANGES_REQUESTED** [review]
Almost there! 💪

**Comment by vannyle** [comment]
Thanks for the comments! @bolotskydev

**Review by danieltprice — COMMENTED** [review]
LGTM.

**Pull request merged**